solutions/swb-reference/src/services/sshKeyService.ts

```diff
@@ -42,8 +42,44 @@ export default class SshKeyService implements SshKeyPlugin {
   public async listUserSshKeysForProject(
     request: ListUserSshKeysForProjectRequest
   ): Promise<ListUserSshKeysForProjectResponse> {
-    // TODO: implement
-    throw new Error('Method not implemented.');
+    const { projectId, userId } = request;
+
+    // get envMgmtRoleArn and externalId from project record
+    const { envMgmtRoleArn, externalId } = await this._projectService.getProject({ projectId });
+
+    // get EC2 client
+    const { ec2 } = await this._getEc2ClientsForHostingAccount(
+      envMgmtRoleArn,
+      'ListForProject',
+      externalId,
+      this._aws
+    );
+
+    // get ssh key from ec2
+    let keyPairs = [];
+    const sshKeyId = this._getSshKeyId(userId, projectId);
+    try {
+      const ec2DescribeKeyPairsParam = {
+        Filters: [{ Name: 'key-name', Values: [sshKeyId] }],
+        IncludePublicKey: true
+      };
+      const response: DescribeKeyPairsCommandOutput = await ec2.describeKeyPairs(ec2DescribeKeyPairsParam);
+      keyPairs = response.KeyPairs || [];
+    } catch (e) {
+      throw new Ec2Error(e);
+    }
+
+    const res: ListUserSshKeysForProjectResponse = { sshKeys: [] };
+    keyPairs.forEach((key) => {
+      res.sshKeys.push({
+        publicKey: key.PublicKey!,
+        createTime: key.CreateTime!.toISOString(),
+        projectId,
+        sshKeyId,
```

Comment:
since this `forEach` could be going over multiple key pairs, assign `sshKeyId: key.KeyName!`